### PR TITLE
Version updated to nightly

### PR DESCRIPTION
**Pull request by mgandharva**
# Description
quay image tag changed to nightly

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/1435|

# Checklist:

- [x] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [x] I have verified that new and existing unit tests pass locally with my changes
- [x] I have not allowed coverage numbers to degenerate
- [x] I have maintained at least 90% code coverage
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have maintained backward compatibility

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

- [x] Powerflex replication E2E
image (7)
image (8)

- [x]  Powerflex resiliency E2E
image (9)
image (10)

- [x] Powerflex Standalone E2E
image (11)

- [x] Powermax Standalone E2E
image

- [x] Powermax Replication E2E
image

- [x] Powermax Resiliency E2E
image

- [x] Unity Standalone E2E
image

- [x]  Powerscale resiliency
image

- [x] Powerscale replication
image

-[x] Powerscale Auth
Screenshot 2024-10-25 130145

- [x]  Powerstore Standalone
image

- [x] Powerstore Resiliency
image

### CODE REVIEW

File: config/samples/storage_v1_csm_powerflex.yaml [hunk 1]
```diff
@@ -55,7 +55,7 @@ spec:
       - name: snapshotter
         image: registry.k8s.io/sig-storage/csi-snapshotter:v8.1.0
       - name: csi-metadata-retriever
-        image: quay.io/dell/container-storage-modulescsi-metadata-retriever:v1.8.0
+        image: quay.io/dell/container-storage-modules/csi-metadata-retriever:v1.8.0
```
Comment (harishp8889) [inline config/samples/storage_v1_csm_powerflex.yaml:58]: Can we check if there is a nightly image for this also?

File: operatorconfig/driverconfig/powermax/v2.12.0/node.yaml [hunk 2]
```diff
@@ -89,7 +89,7 @@ spec:
             capabilities:
               add: ["SYS_ADMIN"]
             allowPrivilegeEscalation: true
-          image: quay.io/dell/container-storage-modules/csi-powermax:v2.12.0
+          image: quay.io/dell/container-storage-modules/csi-powermax:nightly
           imagePullPolicy: IfNotPresent
```
Comment (harishp8889) [inline operatorconfig/driverconfig/powermax/v2.12.0/node.yaml:93]: Please set the pull policy to 'Always' for all the nightly images

File: operatorconfig/moduleconfig/resiliency/v1.11.0/container-powerstore-node.yaml [hunk 3]
```diff
@@ -19,7 +19,7 @@ securityContext:
   capabilities:
     add: ["SYS_ADMIN"]
   allowPrivilegeEscalation: true
-image: quay.io/dell/container-storage-modules/podmon:v1.11.0
+image: quay.io/dell/container-storage-modules/podmon:nightly
 imagePullPolicy: IfNotPresent
 args:
```
Comment (harishp8889) [inline operatorconfig/moduleconfig/resiliency/v1.11.0/container-powerstore-node.yaml:24]: Along with all these changes, Please do reverify and update the minimal manifests e2e test files where it contains images and etc. And make sure it aligns with the actual one present in the samples folder

**Review by harishp8889 — CHANGES_REQUESTED** [review]

**Comment by francis-nijay** [comment]
@mgandharva Please add TODO comment as to when this night will be changed to v2.12 tag

File: operatorconfig/driverconfig/powermax/v2.12.0/node.yaml (hunk 2 above)
Comment (mgandharva) [reply operatorconfig/driverconfig/powermax/v2.12.0/node.yaml]: updated

**Comment by mgandharva** [comment]
> @mgandharva Please add TODO comment as to when this night will be changed to v2.12 tag

@francis-nijay TODO comments are added. Please review it once.

File: config/samples/storage_v1_csm_powerflex.yaml (hunk 1 above)
Comment (mgandharva) [reply config/samples/storage_v1_csm_powerflex.yaml:58]: Covered  here : https://github.com/dell/csm-operator/pull/762

File: operatorconfig/moduleconfig/resiliency/v1.11.0/container-powerstore-node.yaml (hunk 3 above)
Comment (mgandharva) [reply operatorconfig/moduleconfig/resiliency/v1.11.0/container-powerstore-node.yaml:24]: Addressed

**Review by harishp8889 — APPROVED** [review]
lgtm

**Review by meggm — APPROVED** [review]

**Pull request merged**